vendor/kubevirt.io/kubevirt/tests/utils.go

```diff
@@ -2274,7 +2274,7 @@ func SkipIfNoSriovDevicePlugin(virtClient kubecli.KubevirtClient) {
 }
 
 func SkipIfNoMultusProvider(virtClient kubecli.KubevirtClient) {
-	_, err := virtClient.ExtensionsV1beta1().DaemonSets(metav1.NamespaceSystem).Get("kube-multus-ds-amd64", metav1.GetOptions{})
+	_, err := virtClient.ExtensionsV1beta1().DaemonSets(metav1.NamespaceSystem).Get("kube-multus-amd64", metav1.GetOptions{})
```

Comment:
mm you cannot change files under the vendor directory, next version update will remove it
I think it better to duplicate this method under your code